server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/Serializer.scala

```diff
@@ -53,4 +52,8 @@ class Serializer {
     (m: Map[Id, Any]) => {
       JsObject(m.map { case (k, v) => (k.value, vr.writes(v)) }.toSeq)
     }
+
+  def serialize(session: Session): String = {
+    Json.stringify(sessionWrites.writes(session))
```

Comment:
`Json` object is supposed to find the right `Writes` from implicit scope. you should not pass it here explicitly

Reply:
`Json.stringify(Json.toJson(session))` does this correct?

Reply:
I don't know Json well, if it works for you, that's ok